src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/help-mablBranch.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    mabl will run tests using this branch name
```

Comment:
Perhaps we should clarify this is a `mabl` branch, not an SCM branch?

Reply:
Absolutely, thank you.